src/code/z_play_hireso.c

```diff
@@ -1,27 +1,1360 @@
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 2, BOMBERS_NOTEBOOK_EVENT_RECEIVED_GAROS_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(18, 0)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 2, BOMBERS_NOTEBOOK_EVENT_ESCORTED_CREMIA,
+                               CLOCK_TIME(18, 0), CLOCK_TIME(21, 0)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 3, BOMBERS_NOTEBOOK_EVENT_RECEIVED_GAROS_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(18, 0)),
+        BOMBERS_NOTEBOOK_ENTRY_END,
+    },
+    {
+        /* Shiro */
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 1, BOMBERS_NOTEBOOK_EVENT_RECEIVED_STONE_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 2, BOMBERS_NOTEBOOK_EVENT_RECEIVED_STONE_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 3, BOMBERS_NOTEBOOK_EVENT_RECEIVED_STONE_MASK,
+                               CLOCK_TIME(6, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY_END,
+    },
+    {
+        /* Guru-Guru */
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 1, BOMBERS_NOTEBOOK_EVENT_RECEIVED_BREMEN_MASK,
+                               CLOCK_TIME(18, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY(BOMBERS_NOTEBOOK_ENTRY_POS_CENTER, 2, BOMBERS_NOTEBOOK_EVENT_RECEIVED_BREMEN_MASK,
+                               CLOCK_TIME(18, 0), CLOCK_TIME(5, 59)),
+        BOMBERS_NOTEBOOK_ENTRY_END,
+    },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Draw.s")
+s16 sBombersNotebookDayRectRectLeft[] = { 120, 120, 270, 420 };
+TexturePtr sBombersNotebookDayTextures[] = {
+    gBombersNotebookDay1stTex,
+    gBombersNotebookDay2ndTex,
+    gBombersNotebookDayFinalTex,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016F4EC.s")
+#define DEFINE_EVENT(_enum, icon, _colorFlag, _description, _completedMessage, _completedFlag) icon,
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Update.s")
+s32 sBombersNotebookEventIcons[] = {
+#include "tables/bombers_notebook/event_table.h"
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Init.s")
+#undef DEFINE_EVENT
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/BombersNotebook_Destroy.s")
+s32 sBombersNotebookEventIconWidths[] = { 16, 24, 32 };
+s32 sBombersNotebookEventIconHeights[] = { 16, 28, 28 };
+
+#define DEFINE_EVENT(_enum, _icon, colorFlag, _description, _completedMessage, _completedFlag) colorFlag,
+
+u16 sBombersNotebookEventColorWeekEventFlags[] = {
+#include "tables/bombers_notebook/event_table.h"
+};
+
+#undef DEFINE_EVENT
+
+void BombersNotebook_TextureRectangle(Gfx** gfxP, s32 rxl, s32 ryl, s32 rxh, s32 ryh, s32 tile, s32 s, s32 t, s32 dsdx,
```

Comment:
```suggestion
void BombersNotebook_DrawTexRectScis(Gfx** gfxP, s32 rxl, s32 ryl, s32 rxh, s32 ryh, s32 tile, s32 s, s32 t, s32 dsdx,
```
At least for all the functions in parameter/kaleido/file-select that draw TextureRectangles, this is the style Tharo and I have been following. Not sure where to put the `Scis` in there, but the `DrawTexRect` is usually the base. Any thoughts?

Reply:
Seems fine. Though decided to go with `DrawScisTexRect` since that matches the macro better.